app/assets/stylesheets/show_page.scss

```diff
@@ -1,4 +1,55 @@
+$celtics: #195a07;
+
 .meetup-avatar {
-  height: 50px;
-  width: 50px;
+  height: 100%;
+  width: 100%;
+  max-height: 60px;
+  max-width: 60px;
+
+  min-width: 40px;
+  min-height: 40px;
+  transform: translateY(15%);
+}
+
+.show-info-left {
+  max-width: 320px;
+}
+#court-show-info {
+  padding: 15px;
+  width: 100%;
+  background-color: darken($celtics, 3);
+}
+
+#meetups-container {
+  padding: 20px;
+  text-align: left;
+}
+
+#edit-court {
+  text-align: center;
+}
+
+#schedule-game-button {
+  text-align: center;
+}
+
+#showMap {
```

Comment:
Avoid using id selectors<br>Selector `showMap` should be written in lowercase with hyphens